### ISSUE TITLE
tags seem to be case sensitive

### ISSUE THREAD

**Issue by oupala** [totocaster/metalsmith-tags#51]
Tags seem to be case sensitive.

- I create a document with `api` tag
- I create another document with `API` tag

On both document, tag link is `http://myserver/tags/api/`.

But when I click on the link, only the first document appear, the one which have the `api` tag (lowercase).

So tags seem to be partially cas sensitive (the slug is lowercased, but the real tag is API or api). Is there a way to fix that?

**Comment by benpolinsky** [comment]
Agreed. I think this is because metalsmith doesn't allow for two files which differ only by casing to be processed as separate files (correct me if I am wrong).  Though the web spec for URLs technically allows it (aside from domains - google.com vs Google.com) 

It'd be nice to make tags case insensitive, so an 'api' tag is the same as 'Api'.  I'll work on a PR.

**Comment by oupala** [comment]
@benpolinsky Have you had time to work on this PR?
